tracee-rules/signatures/rego/regosig/traceerego.go

```diff
@@ -26,18 +28,25 @@ type RegoSignature struct {
 	selectedEvents []types.SignatureEventSelector
 }
 
-const queryMatch string = "data.main.tracee_match"
-const querySelectedEvents string = "data.main.tracee_selected_events"
-const queryMetadata string = "data.main.__rego_metadoc__"
+const queryMatch string = "data.%s.tracee_match"
+const querySelectedEvents string = "data.%s.tracee_selected_events"
+const queryMetadata string = "data.%s.__rego_metadoc__"
+const packageNameRegex string = `package\s.*`
 
 // NewRegoSignature creates a new RegoSignature with the provided rego code string
 func NewRegoSignature(regoCodes ...string) (types.Signature, error) {
 	var err error
 	res := RegoSignature{}
 	regoMap := make(map[string]string)
 
-	for index, regoCode := range regoCodes {
-		regoModuleName := fmt.Sprintf("rego_%d", index)
+	re := regexp.MustCompile(packageNameRegex)
+
+	var pkgName string
+	for _, regoCode := range regoCodes {
+		regoModuleName := strings.Split(re.FindString(regoCode), " ")[1]
+		if !strings.Contains(regoCode, "package tracee.helpers") {
+			pkgName = regoModuleName
+		}
 		regoMap[regoModuleName] = regoCode
 	}
 
```

Comment:
Shouldn't we check that pkgName is not an empty string (for the case of "package tracee.helpers")?